features/support/fakebin/git

```diff
@@ -4,7 +4,7 @@
 # asserted that they ran.
 
 command="$1"
-[ "$command" = "config" ] || echo git "$@" >> "$HOME"/.history
+[ "$command" = "config" -a "${#@}" -le 2 ] || echo git "$@" >> "$HOME"/.history
```

Comment:
A shorter expression for `${#@}` is `$#`. But, I don't think that we need to log the `git config` statements into `.history` log file. Instead, couldn't we read from `git config` to verify that this configuration is being set? We could add a new custom step that reads like:

```
Then the branch should merge "refs/heads/fixes" from remote "git@github.com:mislav/jekyll.git"
```

and the step could use `git config` internally to extract this information. How does that sound?